### PR TITLE
Handle subscriptions over AIOHTTP WebSockets

### PR DESCRIPTION
**Pull request by Maximilien-R**

### CODE REVIEW

File: examples/aiohttp/dogs/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Tatiflette AIOHTTP - WebSockets subscriptions
+
+This is a fully Dockerised example.
```
Comment (abusi) [inline examples/aiohttp/dogs/README.md]: Can you add that docker and docker-compose are mandatory deps for this example to work ? (Add the version you validate the example with for both of them)

**Comment by abusi** [comment]
Everything else seems fine !
I'm liking it !

File: examples/aiohttp/dogs/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+If you want to test and run it, the only command to run is: `make run`.
+
+`Docker` & `docker-compose` are mandatory to run this example.
+
+Should work with:
+* Docker >= 18.06
+* docker-compose >= 1.22.0
```
Comment (abusi) [inline examples/aiohttp/dogs/README.md]: :+1: :heart:

File: README.md [hunk 3]
```diff
@@ -164,6 +166,13 @@ web.run_app(
 * **executor_http_endpoint**: Endpoint where the GraphQL Engine will be attached, by default on `/graphql`
 * **executor_http_methods**: HTTP Method where the GraphQL Engine will be attached, by default on **POST** and **GET**
 
+### Tartiflette with subscriptions
+
+Tartiflette embed an easy way to deal with subscription. The only thing to do is
```
Comment (achedeuzot) [inline README.md]: embedS

File: examples/aiohttp/dogs/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+redis
```
Comment (achedeuzot) [inline examples/aiohttp/dogs/requirements.txt]: no versions ? :x

**Pull request merged**